Zend/tests/pipe_operator/exception_interruption.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+A pipe interrupted by an exception, to demonstrate correct order of execution.
+--FILE--
+<?php
+
+function foo()     { echo __FUNCTION__, PHP_EOL; return 1; }
+function bar()     { echo __FUNCTION__, PHP_EOL; return false; }
+function baz($in)  { echo __FUNCTION__, PHP_EOL; return $in; }
+function quux($in) { echo __FUNCTION__, PHP_EOL; throw new \Exception('Oops'); } // This is line 6.
```

Comment:
This comment is not useful, since you are not printing the stacktrace. You can `echo $e, PHP_EOL` in the `catch()`.